ospd/command/command.py

```diff
@@ -591,13 +590,13 @@ def handle_xml(self, xml: Element) -> bytes:
             id_.text = scan_id_aux
             return simple_response_str('start_scan', 100, 'Continue', id_)
 
-        scan_process = create_process(
-            func=scan_func, args=(scan_id, scan_target)
-        )
-
-        self._daemon.scan_processes[scan_id] = scan_process
-
-        scan_process.start()
+        if dry_run:
```

Comment:
Shouldn't this be `if not dry_run`?

Reply:
No. It must be "if dry_run". So the dry run will be started immediately.